src/index.html

```diff
@@ -11,9 +11,29 @@
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Document</title>
   <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
```

Comment:
Please define here a font-weights which do you want to use. Now you load only `font-weight: regular` (or `font-weight: 400`). But there are also `medium (500)` and `bold(700)` in the mock
![image](https://user-images.githubusercontent.com/29413892/56794266-713e7a80-6816-11e9-84f9-aec8a3ce989e.png)

Then the link will look like this.
![image](https://user-images.githubusercontent.com/29413892/56794332-98954780-6816-11e9-8f46-6b6bd096227e.png)